src/components/Filters/Filters.tsx

```diff
@@ -96,6 +97,14 @@ const Filters: React.FC<FiltersProps> = ({
     }
   }, [filtersValues, fields]);
 
+  const validateFieldValues = () => {
+    const validationErrors = [];
+    // TO DO: check validation for individual fields values
+    setHasInvalidValues(!!validationErrors.length);
+
+    return validationErrors;
```

Comment:
Pls, what was the intention with returning of the array of errors? Can be removed?